dll/win32/msvcrt/CMakeLists.txt

```diff
@@ -27,7 +35,7 @@ target_link_libraries(msvcrt crt wine ${PSEH_LIB})
 if(MSVC)
     # export of deleting destructor "name"
     add_target_link_flags(msvcrt "/ignore:4102")
-    add_target_property(libmsvcrt STATIC_LIBRARY_FLAGS "/ignore:4102")
+    add_target_property(libmsvcrt_real STATIC_LIBRARY_FLAGS "/ignore:4102")
```

Comment:
Change to `STATIC_LIBRARY_OPTIONS` while we're here?

Reply:
Oh yeah. And target_link_options also